app/code/Magento/Wishlist/view/frontend/web/js/add-to-wishlist.js

```diff
@@ -19,9 +19,19 @@ define([
             qtyInfo: '#qty',
             actionElement: '[data-action="add-to-wishlist"]'
         },
+        defaultParamsData: {},
 
         /** @inheritdoc */
         _create: function () {
+            var self = this;
+            // Init default params data
+            $('[data-action="add-to-wishlist"]').each(function (index, element) {
```

Comment:
you can use `$(self.options.actionElement)` instead of  `$('[data-action="add-to-wishlist"]')`?

Reply:
Thanks for your suggest.